src/screens/SendReceive/Send/DestinationScreen.tsx

```diff
@@ -88,6 +94,32 @@ const DestinationScreen = ({ navigation, style, route: { params } }: ScreenProps
     }
   }
 
+  const handleContactPress = (contactId: Contact['id']) => {
+    const contact = contacts.find((c) => c.id === contactId)
+
+    if (contact) {
+      posthog?.capture('Send: Selected contact to send funds to')
+
+      navigation.navigate('SendNavigation', {
+        screen: nextScreen,
+        params: { toAddressHash: contact.address }
+      })
+
+      closeContactSelectModal()
+    }
+  }
+
+  const handleAddressPress = (toAddressHash: AddressHash) => {
+    posthog?.capture('Send: Selected own address to send funds to')
+
+    navigation.navigate('SendNavigation', {
+      screen: nextScreen,
+      params: { toAddressHash }
+    })
```

Comment:
For addresses and contacts: let's not directly navigate to the next screen on select.

As a user, I find it important to be able to check directly if the right address has been selected (it's easy to do an accidental tap).

Idea: would be to create a little animation in the "destination address" field, when it's automatically filled up. This would attract the user's attention and encourage them to quickly check the address hash.

Do you think you could create such an animation? Don't take too much time for this as I may tweak it : for now, let's just make the input's background smoothly fade to the accent color and back to the original color (1s total). WDYT? :)

Reply:
I've pushed a commit! I used the `global.pale` color cause I didn't like the animation with the `bg.accent` one. It looks weird, I think because of the opacity level. Feel free to tweak it.

Reply:
Yeah I expected that! Good choice.